pypytools/util.py

```diff
@@ -22,6 +23,8 @@ def clonefunc(f):
             co.co_firstlineno, co.co_lnotab, co.co_freevars, co.co_cellvars]
     if PY3:
         args.insert(1, co.co_kwonlyargcount)
+    if PY3 and PY3M >= 8:
+        args.insert(1, co.co_posonlyargcount)
```

Comment:
Could you also add this to fix 3.11 and above? That would fix #4
```py
    if PY3 and PY3M >= 11:
        args.insert(12, co.co_qualname)
        args.insert(15, co.co_exceptiontable)
```

Reply:
okay added it